app/experimenter/experiments/constants.py

```diff
@@ -76,6 +75,14 @@ class ExperimentConstants(object):
         ("70.0", "Firefox 70.0"),
```

Comment:
I guess we might want to tack a few more versions on there while we're doing this since we're about to hit 70.  Maybe do up to 80 and then we don't need to think about it again for a little while?

Reply:
sounds good!